client/Brocfile.js

```diff
@@ -1,11 +1,23 @@
-/* global require, module */
-
+/* global require, module, process */
+var env = process.env.EMBER_ENV;
```

Comment:
I would guess there probably is a better way, I was just following the examples here: https://www.npmjs.com/package/ember-cli-inline-content .  This approach seems to work for now so I figured we can circle back if we don't have a better mechanism to use now.  I'm wide open to suggestion however.